requests/_bug.py

```diff
@@ -0,0 +1,99 @@
+"""Module containing bug report helper(s)."""
+from __future__ import print_function
```

Comment:
I'm not a huge fan of the `_` used here, for the module name. 

I'd rather see `python -m requests.report` or something maybe.